specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/preview/2018-06-30-preview/guestconfiguration.json

```diff
@@ -852,6 +1013,323 @@
         }
       },
       "description": "Reason and code for the compliance of the guest configuration assignment resource."
+    },
+    "SoftwareUpdateProfile": {
```

Comment:
id, name and type top level properties missing. Did you miss a reference to ProxyResource?

Reply:
Referenced from Proxy now

Reply:
Added Proxy reference now